api/external/applications/applications.proto

```diff
@@ -211,59 +389,90 @@ message GetDeleteDisconnectedServicesConfigReq{}
 message UpdateDeleteDisconnectedServicesConfigRes {}
 
 message Service {
+  // The Chef Habitat Supervisor ID.
   string supervisor_id = 1;
+  // Combination of the service version and release in a single string like:
```

Comment:
This looks a bit odd in the UI

![Screen Shot 2019-12-10 at 4 51 37 PM](https://user-images.githubusercontent.com/22962/70581817-5fbf9e00-1b6d-11ea-8d30-ae41b2d010e3.png)
```suggestion
  // Combination of the service version and release in a single string.
```